presto-main/src/main/java/com/facebook/presto/operator/BigintGroupByHash.java

```diff
@@ -397,7 +397,7 @@ public boolean process()
             checkState(lastPosition < positionCount, "position count out of bound");
             checkState(!finished);
 
-            // needRehash() == false indicates we have reached capacity boundary and a rehash is needed.
+            // needRehash() == true indicates we have reached capacity boundary and a rehash is needed.
```

Comment:
Extract to separate preparatory commit. (this & same above)

Reply:
ok